agents/exec/src/main/resources/org.eclipse.che.exec.script.sh

```diff
@@ -23,7 +32,7 @@ if [ ${CURL_INSTALLED} = false ] && [ ${WGET_INSTALLED} = false ]; then
   CURL_INSTALLED=true
 fi
 
-test "$(id -u)" = 0 || SUDO="sudo -E"
+if is_current_user_root && is_current_user_sudoer; then SUDO="sudo -E"; fi
```

Comment:
Ok your version is more consistent with the rest of the script. Will change that